### DISCUSSION TITLE
how to change model status on view action?

### DISCUSSION THREAD

**Discussion by devahmd** [filamentphp/filament#13516] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

I created a contact messages resource, I need to set the is_read column to true when the user click on view action, and thank you.
I already read the docs but didn't find anything related to that.

**Comment by leandrocfe** [comment]
ViewPage is a LW component, so you can use the mount method

```php
public function mount(int|string $record): void
{
    parent::mount($record);

    $this->getRecord()->update(['is_read' => true]);
}
```

**Reply by devahmd** [reply]
Im using a modal, not a page

**Reply by leandrocfe** [reply]
```php
->mountUsing(function($record){
$record->update(['is_read' => true]);
})
```
?
